babel/test/evaluate.js

```diff
@@ -0,0 +1,64 @@
+import test from 'ava';
+import * as _ from 'lodash';
+import {testOrbs} from './exampleMatchesData';
+import {testCombined} from './exampleMatchesData';
+import {testMatchData} from './exampleMatchesData';
+import {evaluate} from '../src/board';
+import {findMatches} from '../src/board';
+import {combineMatches} from '../src/board';
+import {Board} from '../src/board';
+let board = new Board(5, 5); 
```

Comment:
I did not use `test.before()` here because the `board.orbs` assignment below was not being recognized. I'm not at all sure why.